### DISCUSSION TITLE
Not able to access Supabase API

### DISCUSSION THREAD

**Discussion by paulduplantis** [supabase/supabase#31308] [Questions]
I am trying to pull records from the Supabase API into an HTML file through a separate Javascript file but have not been able to make it work. I executed the following curl command in Powershell to verify the API and received the following error. Everything seems to be setup correctly in Supabase but I must be missing something. Any ideas?.......

 curl.exe -X GET "https://qednuirrccgrlcqrszmb.supabase.co/rest/v1/subject_explorer_records" -H "Authorization: Bearer API key inserted here" -H "Content-Type: application/json"
{"message":"No API key found in request","hint":"No `apikey` request header or url param was found."}

**Comment by GaryAustin1** [comment]
You need an apikey header with just the anon or service_role jwt/key in it also.
See the API docs section of the dashbaord (use the BASH option).

**Reply by paulduplantis** [reply]
Oh cool. Let me check that out. Much appreciated!!
